src/ci/docker/run.sh

```diff
@@ -236,6 +236,7 @@ docker \
   --env TOOLSTATE_REPO \
   --env TOOLSTATE_PUBLISH \
   --env CI_JOB_NAME="${CI_JOB_NAME-$IMAGE}" \
+  --env COMPILETEST_NEEDS_ALL_LLVM_COMPONENTS \
```

Comment:
Hm, I think this just forwards the environment, but it looks like it's not actually getting set anywhere? I would expect this bit to not be needed, because we can set this in src/ci/run.sh which should be run on all platforms; that script runs inside the docker container on Linux.

Reply:
I misinterpreted what this command does.
Moved the variable setting to `src\ci\run.sh`.